README.md

```diff
@@ -1,20 +1,45 @@
-# Universal Data Plane API Working Group (UDPA-WG)
+# xDS API Working Group (xDS-WG)
 
 # Goal
 
-The objective of the Universal Data Plane API Working Group (UDPA-WG) is to bring together parties across the industry interested in a common control and configuration API for data plane proxies and load balancers. 
+The objective of the xDS API Working Group (xDS-WG) is to bring together parties
+across the industry interested in a common control and configuration API for
+data plane proxies and load balancers, based on the xDS APIs.
 
 # Vision
 
-The vision of the Universal data Plane API (UDPA) is articulated at [https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a). We will pursue a set of APIs that provide the de facto standard for L4/L7 data plane configuration, similar to the role played by OpenFlow at L2/L3/L4 in SDN.
+The xDS vision is one of a universal data plane API, articulated at
+[https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a).
+xDS aims to provide a set of APIs that provide the de facto standard for L4/L7
+data plane configuration, similar to the role played by OpenFlow at L2/L3/L4 in
+SDN.
 
-The APIs will be defined canonically in proto3 and incrementally evolve from [existing Envoy xDS APIs](https://github.com/envoyproxy/data-plane-api) via a well defined [stable API versioning policy](https://docs.google.com/document/d/1xeVvJ6KjFBkNjVspPbY_PwEDHC7XPi0J5p1SqUXcCl8/edit#). APIs will cover service discovery, load balancing assignments, routing discovery, listener configuration, secret discovery, load reporting, health check delegation, etc.
+The [existing Envoy xDS
+APIs](https://github.com/envoyproxy/envoy/tree/master/api)
+constitute the basis for this vision and will incrementally evolve towards
+supporting a goal of client neutrality. The xDS APIs have two components, a
```

Comment:
I suggest swapping the last two sentences of this paragraph with the text in the next paragraph.  That way, all of the discussion of client-neutrality will be in one place.